packages/vaadin-lumo-styles/src/components/message.css

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright (c) 2000 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+:host {
+  display: flex;
+  flex-direction: row;
+  outline: none;
+  color: var(--lumo-body-text-color);
+  font-family: var(--lumo-font-family);
+  font-size: var(--lumo-font-size-m);
+  line-height: var(--lumo-line-height-m);
+  padding: var(--lumo-space-s) var(--lumo-space-m);
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  -webkit-text-size-adjust: 100%;
+  --_focus-ring-color: var(--vaadin-focus-ring-color, var(--lumo-primary-color-50pct));
+  --_focus-ring-width: var(--vaadin-focus-ring-width, 2px);
+}
+
+:host([hidden]) {
+  display: none !important;
+}
+
+[part='content'] {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+[part='header'] {
+  align-items: baseline;
+  display: flex;
+  flex-flow: row wrap;
+  min-height: calc(var(--lumo-font-size-m) * var(--lumo-line-height-m));
+}
+
+[part='name'] {
+  font-weight: 500;
+  margin-right: var(--lumo-space-s);
+}
+
+[part='name']:empty {
+  margin-right: 0;
+}
+
+[part='message'] {
+  white-space: pre-wrap;
+}
+
+::slotted([slot='avatar']) {
+  --vaadin-avatar-outline-width: 0;
+  --vaadin-avatar-size: var(--lumo-size-m);
+  flex-shrink: 0;
+  margin-top: calc(var(--lumo-space-s));
+  margin-inline-end: calc(var(--lumo-space-m));
+}
+
+::slotted(vaadin-markdown) {
+  white-space: normal;
+}
+
+:host([focus-ring]) {
+  box-shadow: inset 0 0 0 var(--_focus-ring-width) var(--_focus-ring-color);
+}
+
+:host([dir='rtl']) [part='name'] {
+  margin-left: var(--lumo-space-s);
+  margin-right: 0;
+}
+
+:host([dir='rtl']) [part='name']:empty {
+  margin-left: 0;
+}
+
+[part='time'] {
+  color: var(--lumo-secondary-text-color);
+  font-size: var(--lumo-font-size-s);
+}
```

Comment:
nit: maybe we could move this selector closer to other `part=*` selectors.

Reply:
Done.